README.rst

```diff
@@ -78,12 +78,16 @@ To export Masks for an Image or Plate::
     # e.g. Export to 1.zarr/labels/A
     $ omero zarr masks Image:1 --label-name=A
 
-The default behaviour is to export all masks on the Image to a single 5D
-"labeled" zarr array, with a different value for each mask Shape.
-An exception will be thrown if any of the masks overlap.
+    # Allow overlapping masks or polygons (overlap will be sum of each label)
```

Comment:
The sum of each label won't necessarily be unique.

Reply:
Yes, that matches the thrid and second and third scenarios I raised in https://github.com/ome/omero-cli-zarr/pull/120#issuecomment-1161990656. Any suggestion on the best way to handle overlaps?

Reply:
What about "set all overlapping regions to value of `len(shapes) + 2` (one more than the last shape)"?

Reply:
I'll give it a try...